src/components/ScrollableCarousel.js

```diff
@@ -122,16 +122,18 @@ export default class ScrollableCarousel extends PureComponent {
             this.state.fullyLeftScrolled
               ? "opacity-0 pointer-events-none"
               : "opacity-100",
-            "absolute cursor-pointer select-none -left-2 top-1/2 transform -translate-y-1/2 z-10 transition-all duration-250"
+            "absolute select-none -left-2 top-1/2 transform -translate-y-1/2 z-10 transition-all duration-250"
```

Comment:
Having `cursor-pointer` here would cause the pointer cursor to come up well outside of the circle.  Moving it to the child component seems to make it look better.